libs/SalesforceOAuth/SalesforceOAuth/Classes/SFOAuthCredentials+Internal.h

```diff
@@ -24,7 +24,7 @@
 
 #import "SFOAuthCredentials.h"
 #import <SalesforceSecurity/SFEncryptionKey.h>
-#import <SalesforceCommonUtils/SalesforceCommonUtils.h>
+#import <SalesforceSDKCommon/SalesforceSDKCommon.h>
```

Comment:
Can we move this into `SFOAuthKeychainCredentials.m`?  It's misplaced in this file.